src/js/stores/AppFormStore.js

```diff
@@ -172,16 +178,18 @@ function resolveResponseAttributePathToFieldId(attributePath) {
 }
 
 function populateFieldsFromModel(app, fields) {
-  Object.keys(app).forEach((appKey) => {
+  var paths = Util.detectPathsInObject(app, null, ["env"]);
```

Comment:
I know that, but it feels arbitrary without context. I suggest having a comment that explains the situation: `// Treat "env" as a simple value` or something